src/generator/index.js

```diff
@@ -29,6 +29,11 @@ module.exports = function generator (options = {}) {
     corridorLengthMaximum,
     regularity,
     weightedRoomsTable,
+    randomize,
+    passes,
+    born = [5, 6, 7, 8],
+    survive = [4, 5, 6, 7, 8],
+    topology= 8
```

Comment:
If you don't mind, fix the formatting here please. I can also make this change sometime soon.